web/src/tippyjs.ts

```diff
@@ -426,6 +426,21 @@ export function initialize(): void {
         },
     });
 
+    delegate("body", {
+        target: ".pill-container .pill",
+        appendTo: () => document.body,
+        delay: LONG_HOVER_DELAY,
+        onShow(instance) {
+            const $elem = $(instance.reference)[0].textContent;
+            if ($elem) {
+                instance.setContent($elem);
+            }
+        },
+        onHidden(instance) {
+            instance.destroy();
+        },
+    });
```

Comment:
Can't we directly add `data-tippy-content` attribute to the html element instead of doing this?